tool/tctl/common/plugin/awsic.go

```diff
@@ -0,0 +1,242 @@
+		PlaceHolder("LABELSPEC").
+		SetValue(&p.install.awsIC.userLabels)
+
+	cmd.Flag("group-name", "Add AWS group to group import list by name. Can be a glob, or enclosed in ^$ to specify a regular expression. If no filters are supplied then all AWS groups will be imported.").
+		StringsVar(&p.install.awsIC.groupNameFilters)
+	cmd.Flag("account-name", "Add AWS Account to account import list by name. Can be a glob, or enclosed in ^$ to specify a regular expression. All AWS accounts will be imported if no items are added to account import list.").
+		StringsVar(&p.install.awsIC.accountNameFilters)
+	cmd.Flag("account-id", "Add AWS Account to account import list by ID. All AWS accounts will be imported if no items are added to account import list.").
+		StringsVar(&p.install.awsIC.accountIDFilters)
+}
+
+// InstallAWSIC installs AWS Identity Center plugin.
+func (p *PluginsCommand) InstallAWSIC(ctx context.Context, args installPluginArgs) error {
+	awsICArgs := p.install.awsIC
+	if err := awsICArgs.validate(); err != nil {
+		return trace.Wrap(err)
+	}
+
+	userFilters, err := awsICArgs.parseUserFilters()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	groupFilters, err := awsICArgs.parseGroupFilters()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	accountFilters, err := awsICArgs.parseAccountFilters()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	req := &pluginspb.CreatePluginRequest{
+		Plugin: &types.PluginV1{
+			Metadata: types.Metadata{
+				Name: apicommon.OriginAWSIdentityCenter,
+				Labels: map[string]string{
+					"teleport.dev/hosted-plugin": "true",
+				},
+			},
+			Spec: types.PluginSpecV1{
+				Settings: &types.PluginSpecV1_AwsIc{
+					AwsIc: &types.PluginAWSICSettings{
+						IntegrationName: apicommon.OriginAWSIdentityCenter,
+						Region:          awsICArgs.region,
+						Arn:             awsICArgs.arn,
+						ProvisioningSpec: &types.AWSICProvisioningSpec{
+							BaseUrl: awsICArgs.scimURL.String(),
+						},
+						AccessListDefaultOwners: awsICArgs.defaultOwners,
+						CredentialsSource:       types.AWSICCredentialsSource_AWSIC_CREDENTIALS_SOURCE_SYSTEM,
+						UserSyncFilters:         userFilters,
+						GroupSyncFilters:        groupFilters,
+						AwsAccountsFilters:      accountFilters,
+					},
+				},
+			},
+		},
+		StaticCredentials: &types.PluginStaticCredentialsV1{
+			ResourceHeader: types.ResourceHeader{
+				Metadata: types.Metadata{
+					Labels: map[string]string{
+						"aws-ic/scim-api-endpoint": awsICArgs.scimURL.String(),
+					},
+					Name: types.PluginTypeAWSIdentityCenter,
+				},
+			},
+			Spec: &types.PluginStaticCredentialsSpecV1{
+				Credentials: &types.PluginStaticCredentialsSpecV1_APIToken{
+					APIToken: awsICArgs.scimToken,
+				},
+			},
+		},
+	}
+
+	text, _ := protojson.MarshalOptions{Multiline: true, Indent: "  "}.Marshal(req)
```

Comment:
Any particular reason to ignore the error?

Reply:
:eyeroll: Leftover debug printf